eap6-support/hawkular-wildfly-agent-feature-pack-eap6/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml

```diff
@@ -622,11 +623,11 @@
         <operation-dmr name="Shutdown" internal-name="shutdown" > <!-- shutdown and restart -->
```

Comment:
@theute also said he thinks the power ops should be disabled (see comment in https://issues.jboss.org/browse/HAWKULAR-1183) so I will do that before merging this.